scylladb/scylla
# doc: create migration redirections

## Motivation

According to the [migration plan](https://docs.google.com/document/d/15yBf39j15hgUVvjeuGR4MCbYeArqZrO1ir-z_1Urc6A/edit?usp=sharing), the domain docs.scylladb.com will point to this repository. 

The original repository (scylladb/scylla-docs) did not have multi-version enabled, while this repository does:

- Before:  docs.scylladb.com/getting-started.
- Now: docs.scylladb.com/**stable**/getting-started/

For this reason, we should create redirects for all the links that do not have the version name.

This PR edits the `redirects.yml` file as follows:

- Creates redirections for all the files that we have migrated in https://github.com/scylladb/scylla/pull/10976, adding the "stable" prefix.
- Moves the contents from the original `redirections.yml` file to the new `redirects.yml` file taking into account multiversion.

## How to test this PR

1. Enter the docs folder.
2. Run `make redirects`.
3. Open `docs/_build/dirhtml/kb/index.html` in your browser. It should redirect you to `stable/kb`

Before we change the domain name, we will do additional tests on the live site once this PR gets merged.
